snapshot.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-18.6
+resolver: lts-18.10
```

Comment:
Apparently this will break HLS until they cut a release that supports GHC 8.10.7. There's nothing critical in the newer LTS, so we can stick with 18.6 (which is the latest LTS for GHC 8.10.4).

Reply:
```suggestion
resolver: lts-18.6
```